qa/rpc-tests/mempool_size_limit.py

```diff
@@ -8,12 +8,17 @@
 from test_framework.authproxy import JSONRPCException
 from test_framework.util import assert_equal, initialize_chain_clean, \
     start_nodes, sync_blocks, sync_mempools, connect_nodes_bi, mark_logs,\
-    get_epoch_data, swap_bytes
+    get_epoch_data, hex_str_to_bytes, swap_bytes, download_snapshot
+#from test_framework.script import CScript, CScriptNum, OP_CHECKBLOCKATHEIGHT, OP_RETURN, OP_PUSHDATA1, OP_PUSHDATA2, OP_1
```

Comment:
Can we get rid of those unused `import`s?

Reply:
Yes, good point, I moved them next to the `create_tx_script_noraw()` implementation (where they would be needed), which is commented out and left for future reference.